test/test_unistd.rs

```diff
@@ -104,6 +104,57 @@ mod linux_android {
     }
 }
 
+#[test]
+#[cfg(not(any(target_os = "ios", target_os = "macos")))]
```

Comment:
Why aren't these run on these targets?

Reply:
Please leave that as a comment in the code here for both this and the following test.